test/unit/test_log_newline.py

```diff
@@ -0,0 +1,19 @@
+import os
+import platform
+from pathlib import Path
+
+import pytest
+
+from exasol_integration_test_docker_environment.lib.logging.command_log_handler import (
+    CommandLogHandler,
+)
+
+
+def test_new_line(tmp_path):
+    log_file_path = Path(tmp_path, "test_cmd_log_handler.log")
+    with CommandLogHandler(log_file_path, None, "test for multi lines") as log_handler:
+        log_handler.handle_log_line("log line 1")
+        log_handler.handle_log_line("log line 2")
+        with open(log_file_path) as log_file:
+            line_count = sum(1 for line in log_file)
```

Comment:
Interesting construction, I like it; this actually would stream the file , which is good for large files. Maybe a bit overkill, but perfectly ok.